locales/de-DE/common.json

```diff
@@ -1097,25 +1071,19 @@
+  "warning!SwitchingTheMosaicoTemplateWorks": "<0>Warnung!</0> Ein Wechsel des Mosaico-Templats funktioniert nur, wenn das ursprüngliche un das neue Template die gleichen Felder verwenden. Wenn das nicht der Fall ist, werden Daten VERLOREN gehen. Bitte stelle vor dem Fortfahren sicher, dass deine Daten gesichert wurden.",
   "mergeTagsAreTagsThatAreReplacedBefore": "\n<1>Variablennamen werden vor dem Nachrichtenversand ersetzt.Format: <1>[VARIABLEN_NAME]</1> or <3>[VARIABLEN_NAME/Ersatz]</3>; <5>Ersatz</5> ist ein optionaler Text, der eingefügt wird, wenn <7>VARIABLEN_NAME</7> leer ist.</1>\n",
   "youCanUseAnyOfTheStandardMergeTagsBelow": "\n<1>Diese Liste beinhaltet die systemweit verfügbaren Variablennamen. Jedem Feld einer Liste ist ebenfalls ein Variablenname zugeordnet, der in der Felddefinition der jeweiligen Liste zu finden ist.</1>\n",
-  "mergeTagsAreTagsThatAreReplacedBefore-1": "\n                    <1>Merge tags are tags that are replaced before sending out the message. The format of the merge tag is the following: <1>{tg('TAG_NAME')}</1>. </1>\n                ",
-  "mergeTagsAreTagsThatAreReplacedBefore-1 - TODO: update line above and then delete this line to mark that the translation has been fixed": "\n                    <1>Merge tags are tags that are replaced before sending out the message. The format of the merge tag is the following: <1>{tg('TAG_NAME')}</1>. </1>\n                ",
-  "theWholeMessageIsInterpretedAsHandlebars": "\n                    <1>The whole message is interpreted as Handlebars template (see <1>http://handlebarsjs.com/</1>). You can use any Handlebars blocks and expressions\n                        in the template. The merge tags form the root context of the Handlebars template.</1>\n                ",
-  "theWholeMessageIsInterpretedAsHandlebars - TODO: update line above and then delete this line to mark that the translation has been fixed": "\n                    <1>The whole message is interpreted as Handlebars template (see <1>http://handlebarsjs.com/</1>). You can use any Handlebars blocks and expressions\n                        in the template. The merge tags form the root context of the Handlebars template.</1>\n                ",
-  "urlThatPointsToTheSubscribePageOfThe": "URL that points to the subscribe page of the first list in the campaign that allows public subscriptions",
-  "urlThatPointsToTheSubscribePageOfThe - TODO: update line above and then delete this line to mark that the translation has been fixed": "URL that points to the subscribe page of the first list in the campaign that allows public subscriptions",
+  "mergeTagsAreTagsThatAreReplacedBefore-1": "\n                    <1>Variablennamen sind Bezeichnungen, welche vor dem Versenden automatisch ersetzt werden. Das Format der Variablennamen ist wie folgt: <1>{tg('TAG_NAME')}</1>. </1>\n                ",
```

Comment:
There should be something telling it, that Merge TAGs are used, suggestion is not good, maybe checking how far the whole MERGE_TAG or TAG is actually visible in the German interface.
```suggestion
  "mergeTagsAreTagsThatAreReplacedBefore-1": "\n                    <1>MERGE_TAG-Variablennamen sind Bezeichnungen, welche vor dem Versenden automatisch ersetzt werden. Das Format der Variablennamen ist wie folgt: <1>{tg('TAG_NAME')}</1>. </1>\n                ",
```